packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItemStyles.ts

```diff
@@ -16,11 +16,10 @@ const useRootStyles = makeStyles({
   base: {
     alignItems: 'center',
     display: 'flex',
+    position: 'relative',
```

Comment:
Out of curiosity, why is `position: 'relative'` needed?

Reply:
This is so that the AvatarGroupItem can receive a `z-index`, only positioned elements can use `z-index`.